pkgs/top-level/all-packages.nix

```diff
@@ -20749,6 +20749,17 @@ in
 
   syncthing-tray = callPackage ../applications/misc/syncthing-tray { };
 
+  syncthingtray-minimal = libsForQt5.callPackage ../applications/misc/syncthingtray { };
+  syncthingtray = libsForQt5.callPackage ../applications/misc/syncthingtray {
+    # Could be qt5.qtwebengine as well
+    webviewProvider = qt5.qtwebkit;
```

Comment:
Also here I guess Qt WebEngine is the saner default by nowadays. Qt WebKit was a thing some time ago, then it has been rotten, then revived and I now consider it finally buried. Maybe I should state my recommendations more clearly in the README.

Reply:
I took these as "recommendations" by reading the `PKGBUILD` (I once was an Arch Linux user) as these seemed to be the first options / suggestions there.